sdk/core/core-sse/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Fixes a bug in NodeJS where we attempt to close the underlying socket despite it being set to null when the stream has been consumed [BUG [#30414](https://github.com/Azure/azure-sdk-for-js/issues/30414)].
```

Comment:
nit: "a bug in NodeJS" sounds like a NodeJS bug. maybe "when running on NodeJS"?

Reply:
addressed in https://github.com/Azure/azure-sdk-for-js/pull/30707/commits/d9a8ea5764e27702ef7c89a621b0373e189e63dc